packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -104,6 +105,7 @@ class MediaQueryData {
     this.disableAnimations = false,
     this.boldText = false,
     this.navigationMode = NavigationMode.traditional,
+    this.gestureSettings = const DeviceGestureSettings(touchSlop: 16.0)
```

Comment:
Can this use kTouchSlop instead of hard-coding 16?

Reply:
Done